proposals/NNNN-steering-wheel-location.md

```diff
@@ -0,0 +1,52 @@
+# Steering wheel location
+
+* Proposal: [NNNN](NNNN-steering-wheel-location.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Review ready**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+This proposal is about adding a new parameter to the `VehicleType` structure to inform the app about the steering wheel location within the vehicle.
+
+## Motivation
+
+Some apps want to use different templates based on the steering wheel location. As an example TEXT_WITH_BUTTONS_AND_GRAPHIC works great for left hand drives as buttons are closer to the driver. For right hand drives apps would want to use GRAPHIC_WITH_TEXT_AND_BUTTONS. Today apps would use the language/region code and **guess** the location of the steering wheel.
```

Comment:
Please change "left hand drives" to "left handed drivers" and "right hand drives" to "right handed drivers,"